docs/getting_started/installation.rst

```diff
@@ -97,17 +97,6 @@ line in the `app/AppKernel.php` file of your project:
 
         $ bower install ./vendor/sonata-project/admin-bundle/bower.json
 
-.. note::
-
-    You must enable translator service in `config.yml`.
-
-    .. code-block:: yaml
-
-        framework:
-            translator: { fallbacks: ["%locale%"] }
-
-    For more information: http://symfony.com/doc/current/translation.html#configuration
-
```

Comment:
Why did you remove that part?

Reply:
because it's also incosistent/redundant:

There's both
```
framework:
    translator: { fallbacks: ["%locale%"] }
```
and
```
framework:
    translator: { fallbacks: [en] }
```
Since I needed to bump the section # anyway, cleaned it up and stuck with the generic case